C-Plus-Plus/ds/Array_mirror_inverse.cpp

```diff
@@ -0,0 +1,59 @@
+// C++ implementation of the approach
+#include<bits/stdc++.h>
+using namespace std;
+
+// Function that returns true if
+// the array is mirror-inverse
+bool mirr_inv(int arr[], int n)
```

Comment:
@TEJASWI-TEJASWI Use proper variables name like `mirr_inv` can be updated to `mirror_inverse`.

Reply:
I have update the variable name .
So please review @iamrajiv

Reply:
@TEJASWI-TEJASWI I have requested more changes.

Reply:
I have resolved the issue.
please have a look on PR-969 @iamrajiv

Reply:
Ihave update the readme file please have a look on PR-970 @iamrajiv